identity-account/Cargo.toml

```diff
@@ -17,16 +17,19 @@ hashbrown = { version = "0.9", features = ["serde"] }
 identity-core = { version = "=0.3.0", path = "../identity-core" }
 identity-credential = { version = "=0.3.0", path = "../identity-credential" }
 identity-did = { version = "=0.3.0", path = "../identity-did" }
-identity-iota = { version = "=0.3.0", path = "../identity-iota" }
+identity-iota = { version = "=0.3.0", path = "../identity-iota", default-features = false }
 itoa = { version = "0.4" }
 log = { version = "0.4", default-features = false }
 once_cell = { version = "1.7", default-features = false, features = ["std"] }
 paste = { version = "1.0" }
-riker = { version = "0.4" }
-serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
+riker = { version = "0.4", optional = true }
+serde = { version = "1.0", default-features = false, features = [
+    "alloc",
+    "derive"
+] }
 slog = { version = "2.7" }
 thiserror = { version = "1.0" }
-tokio = { version = "1.5", default-features = false, features = ["rt", "rt-multi-thread", "sync"] }
+tokio = { version = "1.7", features = ["sync"] }
```

Comment:
was the change to 1.7 required to fix wasm compilation? maybe better to update as a separate PR.

Reply:
in any case, the tokio version in the examples should be kept in sync with this one.

Reply:
No that wasn't required for Wasm, I just upgraded it. But should be a separate PR, I agree.